vite-project-kabinet/src/Footer/footer.style.css

```diff
@@ -3,6 +3,15 @@
 /* footer */
 /*********/
 
+.app_container.menu_open footer {
```

Comment:
Detail: moje zkušenost je taková, že je lepší ve stylech nepoužívat  elementy jako selektory (`div`, `footer`, `main` apod) a raději pokaždé používat css classy. Protože se může stát, že budeš chtít sémantický význam (který nesou elementy jako `footer` nebo `main`) změnit nebo přesunout a když adresuješ přes class-name, tak se to lépe mění.